frontend/simple/views/Mailbox.vue

```diff
@@ -192,6 +192,8 @@
   background-color: #1fc8db;
   border-radius: 20px;
 }
+
+.panel-block a { border-bottom: none; }
```

Comment:
Why not adding the class directly at the two elements `a` that really need this class?

It's readable and we can avoid an unnecessary nesting selector. When you do this, what really happens under the wood is: Look for **all** `a` in the DOM, then look for those that are `.panel-block`'s children.

When we apply a class (`.no-border` that you just created) directly to the element it's way faster to connect those styles to the DOM. In matters of performance this is imperceptible most of the times, specially on simple DOMs like GI. Nevertheless, we should aim for a flat CSS structure for better readability and reusability so we can easily scale :)